src/utils/changeUrlforPager.js

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
As this is a helper function for dealing with the legacy pagination of assessment table / original statements table, i'd suggest to move the file into `client/js/components/statement/assessmentTable/utils/` (utils folder to be created). The according test must be moved, then, too.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/71/commits/d96e420e73955eef91d2c648e98e4abe60e22dd1